### DISCUSSION TITLE
瞎写了一个wordpress插件

### DISCUSSION THREAD

**Discussion by znc15** [lsky-org/lsky-pro#759] [分享]
小白第一次写wordpress的插件，可以将wordpress上的图片上传到lskypro，写的菜，大佬别骂（
不知道有没有一样的项目qwq
https://github.com/znc15/wordpress-lskypro

**Comment by heiyuan0801** [comment]
有的  可以发到兰空论坛  lskypro.com

**Reply by znc15** [reply]
寄，白写了（
